packages/vue/__tests__/Authenticator.spec.tsx

```diff
@@ -2,7 +2,7 @@ import { mount } from '@vue/test-utils';
 import { Authenticator } from '../dist';
 import crypto from 'crypto';
 
-// This is needed for jest to get crypto module needed by nanoid
+// This is needed for vitest to get crypto module needed by nanoid
 Object.defineProperty(global.self, 'crypto', {
   value: {
     getRandomValues: (arr: Array<any>) => crypto.randomBytes(arr.length),
```

Comment:
I've been seeing 

![image](https://user-images.githubusercontent.com/43682783/174451539-e0e2cd8e-920b-438e-a614-d8562164d512.png)

(and this is on me bc I wrote the code)

Reply:
Can you check if this polyfill is still needed, and if so, can you try more scoped type for `Array<any>`?

Reply:
Yeah I can update that so GH actions does not complain